tests/foreman/api/test_contentview.py

```diff
@@ -1032,9 +893,10 @@ def test_cv_associate_rh_custom_spin(self):
         view
 
         """
-        content_view = entities.ContentView(organization=self.org.id).create()
-        content_view.set_repository_ids([self.repo.id])
-        self.assertEqual(len(content_view.read().repository), 1)
+        content_view = entities.ContentView(organization=self.org).create()
+        content_view.set_repository_ids([self.repo.id])  # pylint:disable=E1101
+        content_view = content_view.read()
```

Comment:
Why do we need to create ```content_view``` variable here? I saw a lot of similar occurrences through the code, but there were at least two ```assertEqual``` statements, but for one it doesn't seem so obvious for me.